app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+from app.players.player import Player
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
+
+
+def calculate_team_total_rating(gamers: list) -> Player:
+    return sum([gamer.get_rating() for gamer in gamers])
```

Comment:
Does this function return type Player? Fix type annotation.
Also, add more explicit type annotation for ```gamers```.